doc/i3lock-fancy.1

```diff
@@ -54,7 +54,7 @@ Do not fork i3lock after starting.
 
 .TP
 \fB--\fP
-Must be last option. Set command to use for taking a screenshot. Default is
+Specify a custom command to use for taking a screenshot. This must be the final command-line option. The default is
```

Comment:
"final"?

Reply:
Better than 'last', which carries the connotation 'most recent'. Most native English speakers probably wouldn't care either way.